api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -134,13 +134,15 @@ message HttpConnectionManager {
   string server_name = 10;
 
   // The maximum request headers size for incoming connections. The default max
-  // is 60K, based on default settings for http codecs. For HTTP1, the current
-  // limit set by http_parser is 80K. for HTTP2, the default allowed header
-  // block in nghttp2 is 64K. The max configurable setting is 64K in order to
+  // is 60 KiB, based on default settings for http codecs. For HTTP1, the currentB
```

Comment:
Given the new changes, are these sizes still correct/relevant? Isn't is basically fully configurable now on both codecs or at least up to 96KiB max below? Can you potentially simplify the docs?

Reply:
1. Simplified the docs.
2. Removed the bit on per k/v pair limits, I want to work on adding more thorough and cross-codec regression testing around that before it's documented.